docs/operate/config-reference.mdx

```diff
@@ -1075,6 +1075,50 @@ Because each VM supports a different signer type, the default signer can only su
 </TabItem>
 </Tabs>
 
+## defaultRpcConsensusType
+
+:::info
+
+**The consensus type currently only applies to EVM chains.**
+
+Connections to non-EVM chains only support a single RPC url. If more than one RPC url is specified in [rpcUrls](#chainschain_namerpcurls), only the first one will be used.
```

Comment:
we've recently added fallback provider support to both the cosmwasm and sealevel providers, so this statement can be changed to mention that, and also that quorum provider is still only available on EVM chains (in the section about `rpcConsensusType` as well pls)

Reply:
updated in 163547b5ddb345a7c7b90e69be1d609c571ef8c4